src/datafactory/azext_datafactory/generated/custom.py

```diff
@@ -64,7 +66,7 @@ def datafactory_factory_update(client,
     return client.update(resource_group_name=resource_group_name,
                          factory_name=factory_name,
                          tags=tags,
-                         identity={"type": "SystemAssigned"})
+                         identity=json.loads("{\"type\": \"SystemAssigned\"}"))
```

Comment:
Why not use `{"type": "SystemAssigned"}` directly?

Reply:
codegen hasn't started to support this. The last time I change it manually. let me change it back